code/datums/traits/negative.dm

```diff
@@ -355,11 +355,11 @@
 	
 /datum/quirk/poor_aim/add()
 	var/mob/living/carbon/human/H = quirk_holder
-	H.dna.species.aiminginaccuracy += 25
+	H?.dna?.species?.aiminginaccuracy += 25
```

Comment:
Stuff like this we probably want to runtime

Reply:
Honestly, i think it was just removal of the quirk that runtimed, i just did it to both parts.